check_version_hook.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const rn_version = JSON.parse(fs.readFileSync('package.json', 'utf-8'))[
+  'version'
+];
+const android_version = fs
```

Comment:
javascript can't parse XML out of the box 🙃

Reply:
🤦